ubuntu22.04/precompiled/Dockerfile

```diff
@@ -10,6 +10,7 @@ ENV DRIVER_VERSION=$DRIVER_VERSION
 
 ARG KERNEL_VERSION=5.15.0-60-generic
 ENV KERNEL_VERSION=$KERNEL_VERSION
+ENV USE_OPEN_KERNEL_MODULES=$USE_OPEN_KERNEL_MODULES
```

Comment:
This is an env we will set at runtime, not build time, so I believe we can remove this.
```suggestion

```